x-pack/plugins/ml/public/application/services/ml_api_service/index.ts

```diff
@@ -474,13 +474,13 @@ export function mlApiServicesProvider(httpService: HttpService) {
     },
 
     getVisualizerFieldHistograms({
-      indexPatternTitle,
+      indexPattern,
```

Comment:
Should this be `dataView`? Or just `indexPattern`?

Reply:
The code this ultimately calls on the server side is a pattern of ES index/indices - I changed the server side route and code here to indicate that it only needs an index pattern (and not the name/title of a data view).